ext/standard/tests/strings/str_ends_with.phpt

```diff
@@ -0,0 +1,47 @@
+--TEST--
+str_ends_with() function - unit tests for str_ends_with()
+--FILE--
+<?php
+/* Prototype: bool str_ends_with (string $haystack, string $needle);
+   Description: Determine if $haystack ends with $needle
+*/
+$testStr = "beginningMiddleEnd";
+var_dump(str_ends_with($testStr, "End"));
+var_dump(str_ends_with($testStr, "end"));
+var_dump(str_ends_with($testStr, "en"));
+var_dump(str_ends_with($testStr, $testStr."a"));
+var_dump(str_ends_with($testStr, "a".$testStr));
+var_dump(str_ends_with($testStr, ""));
+var_dump(str_ends_with("", ""));
+var_dump(str_ends_with("", " "));
+var_dump(str_ends_with($testStr, "\x00"));
+var_dump(str_ends_with("\x00", ""));
+var_dump(str_ends_with("\x00", "\x00"));
+var_dump(str_ends_with("a", "\x00"));
```

Comment:
```suggestion
```
(as said previously, this one doesn't add anything over `($testStr, "\x00")` above)